library/CMakeLists.txt

```diff
@@ -113,12 +113,6 @@ if(CMAKE_COMPILER_IS_CLANG)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wmissing-declarations -Wmissing-prototypes -Wdocumentation -Wno-documentation-deprecated-sync -Wunreachable-code")
 endif(CMAKE_COMPILER_IS_CLANG)
 
-if(UNSAFE_BUILD)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-error")
```

Comment:
Blocker (fails CI). This section needs to stay, or at least to add `-Wno-error=cpp` (and IIRC this needs to be added at a different position on the command line for GCC and Clang). Otherwise a build with `MBEDTLS_TEST_NULL_ENTROPY` fails due to a `#warning`. To validate this with GCC, run `tests/scripts/all.sh test_null_entropy`.